### ISSUE TITLE
Support React v16

### ISSUE THREAD

**Issue by ahstro** [fraserxu/react-dropdown#106]
Probably just requires adding the version to the React peer dependency, but I'm too lazy to test 😅

**Comment by Spy-Seth** [comment]
I've tested `react-dropdown` with react v16 in my app. Everything seems to works 🎉 

Could you release a new version of the package to remove the peer dependencies warnings ? 😅 

```
warning "react-dropdown@1.3.0" has incorrect peer dependency "react@^0.14.7 || ^15.0.0-0".
warning "react-dropdown@1.3.0" has incorrect peer dependency "react-dom@^0.14.7 || ^15.0.0-0".
```

**Comment by fraserxu** [comment]
Published to v1.3.1
